website/addons/box/templates/box_node_settings.mako

```diff
@@ -0,0 +1,138 @@
+
+<link rel="stylesheet" href="/static/addons/box/box.css">
```

Comment:
With the addition of `addonsettings.css`, these addon-specific css files should no longer be necessary. `btn-box` should be replaced with `btn-addon`. 

@mfraezz Can you make this change post-merge?